server/datastore/mysql/migrations/tables/20240707134035_AddUUIDToCalendarEvents.go

```diff
@@ -0,0 +1,34 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240707134035, Down_20240707134035)
+}
+
+func Up_20240707134035(tx *sql.Tx) error {
+	// UUID is a 36-character string with the most common 8-4-4-4-12 format, xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+	// Reference: https://en.wikipedia.org/wiki/Universally_unique_identifier#Textual_representation
+	if _, err := tx.Exec(`ALTER TABLE calendar_events ADD COLUMN uuid VARCHAR(36) COLLATE utf8mb4_unicode_ci NOT NULL`); err != nil {
+		return fmt.Errorf("failed to add `uuid` column to `calendar_events` table: %w", err)
+	}
+
+	// Generate UUIDs for existing calendar events, without changing the updated_at timestamp
+	if _, err := tx.Exec(`UPDATE calendar_events SET uuid = UUID(), updated_at = updated_at`); err != nil {
```

Comment:
Is this just for the uniqueness of UUIDs? (Given these existing events will not be watched.)

Reply:
yes, to meet the constraint